src/Mvc/Mvc.Core/src/ContentResult.cs

```diff
@@ -19,12 +19,13 @@ public class ContentResult : ActionResult, IStatusCodeActionResult
     public string? Content { get; set; }
 
     /// <summary>
-    /// Gets or sets the Content-Type header for the response.
+    /// Gets or sets the Content-Type header for the response,
+    /// to be used as <see cref="Microsoft.AspNetCore.Http.Headers.ResponseHeaders.ContentType" />.
```

Comment:
How does this cross reference improve understanding? This property refers to a generic HTTP Content-Type header. ResponseHeaders.ContentType is just another property that refers to the same HTTP header, it's no more or less authoritative. They aren't even used together in the implementation.  An authoritative reference like https://www.rfc-editor.org/rfc/rfc4021.html#section-2.2.5 might be more helpful.

Reply:
Prefer `cref` to `href` (because it shows up in Visual Studio) and prefer to stay on the documentation site (better UX).  I shall use `href` if you insist, please confirm!

Reply:
Referencing ResponseHeaders.ContentType from here isn't useful, they're barely related. I'm not aware of any local resource you could cref that would be useful, so either we don't add any link here, or we add an href to an external resource.

Reply:
You can get to [MediaTypeHeaderValue](https://docs.microsoft.com/en-us/dotnet/api/microsoft.net.http.headers.mediatypeheadervalue) from [ContentType](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.headers.responseheaders.contenttype), which is what you want to use when you need to make a `Content-Type` header.  We could refer to that one directly if you prefer.
Your hyperlink goes to a standard document.  The purpose of a library is to isolate ordinary developers from having to read standard documents that are written for library developers.  Let&rsquo;s be merciful and not make the poor developers read that.

Reply:
These two properties wouldn't be used together in a developer workflow, they're both mainly for assignment and are for people working in different contexts. Please remove the link.